pkg/cluster/internal/create/nodes.go

```diff
@@ -155,7 +155,7 @@ func fixupNode(node *nodes.Node) error {
 	if nodes.NeedProxy() {
 		if err := node.SetProxy(); err != nil {
 			// TODO: logging here
-			return errors.Wrapf(err, "failed to set proxy for node %s", node.Name())
+			return errors.Errorf("failed to set proxy for %s", node.Name())
```

Comment:
could you please explain this change?

Reply:
it's a leftover of the first PR, I force pushed and deleted the history. let me fix it
Thanks for catching it